### ISSUE TITLE
TODO: Add privacy controls

### ISSUE THREAD

**Issue by vot** [vot/great-converto#1]
It would be good to offer a table of 5 most recently converted files.
This would require a way of users being able to keep the video private / unlisted (checked by default).

**Comment by adrianRodriguez123** [comment]
> It would be good to offer a table of 5 most recently converted files
> This would require a way of users being able to keep the video private / unlisted (checked by default).

I don't really understand, if every file has a different name, how are you going to display the 5 most recently converted files . Also, will the users need to login to make the list private?

Last, check your license, it seems deprecated.

**Comment by vot** [comment]
Hi @adrianRodriguez123 - thanks for pointing this out.
I just realised I didn't implement Mongo backing here which would be necessary (all data is only stored in memory right now).

The idea is to have the videos that are still available to be listed if the original uploader has opted in to that. Given the current limitation of not having the appropriate storage for metadata I'll close the issue for now until this can be acted upon (I got a bit overzealous with creating issues this evening).

PS Thanks for pointing out LGPL deprecation :)
